CMSgov/onemac
# OY2 11221 Submission Form: Submit button updates

Story: https://qmacbis.atlassian.net/browse/OY2-11221
Endpoint: https://d2xyo0xkd5e0w1.cloudfront.net/

### Details

Move the submit button to inside the PageTitleBar and add functionality so that it is disabled until the submission fulfills all the requirements.

### Changes
- added the submit button to the PageTitleBar's rightSideContents
- connected the submit button to the form/s submit functionality
- updated the button to use cms design system styles and classes
- updated the dev login button to also use cms design system
- adapted the form default onSubmit (return key) to be enabled/disabled the same as Submit button
- updated the color of the pagetitlebar background to match the Figma
- removed extraneous error messaging
- removed Cancel functionality

### Test Plan

1. Log in as State Submitter Active
2. Use the New Submission button to create submissions of each type
a. Verify the Submit button starts in PageTitleBar and is gray
b. Add required fields and attachments one at a time, verify that submit button stays gray until final requirement is met, then turns blue
c. try removing required data from fields and verify button goes gray and blue accordingly
d. Submit the valid form, verify submission is correct.
